css/mac.css

```diff
@@ -49,6 +50,7 @@ button:active {
 
 .project-button button {
```

Comment:
this should just be .project-button {...}
it inherits from .project (if you made a class like that), but anything you write inside it overrides it

Reply:
you could also just leave this one alone for now. Try adding the .header classes from my comments below first to figure out how it works first.